### ISSUE TITLE
guild sync - AttributeError: 'Args' object has no attribute 'digest'

### ISSUE THREAD

**Issue by tekumara** [guildai/guildai#166]
```
$ guild sync a66b1ddb
Traceback (most recent call last):
  File "/home/ec2-user/.virtualenvs/actrain/bin/guild", line 8, in <module>
    sys.exit(main())
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/main_bootstrap.py", line 40, in main
    guild.main.main()
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/main.py", line 33, in main
    _main()
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/main.py", line 40, in _main
    main_cmd.main(standalone_mode=False)
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/external/click/core.py", line 759, in __call__
    return self.main(*args, **kwargs)
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/external/click/core.py", line 714, in main
    rv = self.invoke(ctx)
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/external/click/core.py", line 1132, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/external/click/core.py", line 951, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/external/click/core.py", line 552, in invoke
    return callback(*args, **kwargs)
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/click_util.py", line 141, in fn
    return fn0(*(args + (Args(**kw),)))
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/commands/sync.py", line 70, in sync
    sync_impl.main(args)
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/commands/sync_impl.py", line 31, in main
    runs = runs_impl.runs_for_args(args)
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/commands/runs_impl.py", line 120, in runs_for_args
    filtered = filtered_runs(args, force_deleted, ctx)
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/commands/runs_impl.py", line 128, in filtered_runs
    filter=_runs_filter(args, ctx),
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/commands/runs_impl.py", line 147, in _runs_filter
    _apply_sourcecode_digest_filter(args, filters)
  File "/home/ec2-user/.virtualenvs/actrain/lib/python3.6/site-packages/guild/commands/runs_impl.py", line 234, in _apply_sourcecode_digest_filter
    if args.digest:
AttributeError: 'Args' object has no attribute 'digest'

$ guild --version
guild 0.7.0.rc9
```

**Comment by gar1t** [comment]
Thank you! This is fixed and will be available in the next release (0.7.0.rc10).

**Comment by gar1t** [comment]
Should be fixed in 0.7.0. If you're still seeing this behavior, please feel free to re-open. Thanks again!
